jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -464,6 +537,13 @@ def calculate_photometry(self, dataset=None, aperture=None, background=None,
                    'data_label', 'subset_label', 'timestamp'],
             indexes=[1, 1, 1, 1, 3, 3, 3, 3, 3, 3, 18, 18, 18])
 
+        if self.config == "cubeviz":
+            if data.ndim > 2:
+                slice_val = self._cube_wave
+            else:
+                slice_val = u.Quantity(np.nan, self._cube_wave.unit)
+            phot_table.add_column(slice_val, name="slice_wave", index=29)

```

Comment:
is this going to conflict when we add display-unit support in cubeviz that allows changing between wavelength and frequency units?  We probably can defer and change the name (and possibly handle needing to move units to its own column) at that point, but we'll need a follow-up ticket/issue.

Reply:
This won't be the only conflict when the unit changing stuff happens (flux conversion, anyone?). So yeah, can be dealt with later because a lot of other stuff would also need to change together with it.

Reply:
(Unfortunate though because QTable is supposed to make storing Quantity easier.)

Reply:
ok, since this is getting in before that, I made a ticket for handling all of that linking back here [:cat:](https://jira.stsci.edu/browse/JDAT-4163)